include/igl/flip_edge.cpp

```diff
@@ -72,20 +72,20 @@ IGL_INLINE void igl::flip_edge(
   assert(E(e_32, 0) == v3);
   assert(E(e_32, 1) == v2);
 
-  const size_t ue_24 = EMAP[e_24];
-  const size_t ue_41 = EMAP[e_41];
-  const size_t ue_13 = EMAP[e_13];
-  const size_t ue_32 = EMAP[e_32];
+  const size_t ue_24 = EMAP.coeff(e_24);
```

Comment:
I think these EMAP[...] should all be curved parenthesis. We should use the regular () with bound checking so we can get asserts in debug mode.